.buildkite/engineer

```diff
@@ -13,7 +13,7 @@ fi
 if ! type "engineer" > /dev/null; then
     # Setup Prisma engine build & test tool (engineer).
     set -e
-    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/1.56/latest/$OS/engineer.gz" --output engineer.gz
+    curl --fail -sSL "https://prisma-engineer.s3-eu-west-1.amazonaws.com/linux-static/latest/$OS/engineer.gz" --output engineer.gz
```

Comment:
So this is just for a test?
We probably want instead once we decide to merge all Prisma 5 PRs to use a version?

Reply:
yes, this will be replaced with 1.57 as soon as the engineer pr is merged